plugins/modules/nxos_vrf.py

```diff
@@ -26,6 +26,10 @@
 extends_documentation_fragment:
 - cisco.nxos.nxos
 short_description: Manages global VRF configuration.
+deprecated:
+  alternative: nxos_vrf_global
+  why: Updated modules released with more functionality.
```

Comment:
```suggestion
  why: Updated module released with more functionality.
```

Reply:
had to remove this, this gives an error, just looked it up and seems like no collections have it, nxos has it but for very old modules only